webviews/common/label.tsx

```diff
@@ -14,18 +14,8 @@ export interface LabelProps {
 
 export function Label(label: ILabel & { canDelete: boolean; isDarkTheme: boolean; children?: ReactNode}) {
 	const { name, canDelete, color } = label;
-	const labelColor = gitHubLabelColor(color, label.isDarkTheme, false);
+	const labelColor = gitHubLabelColor(color, label.isDarkTheme, false); /* TODO: Colors */
 	return (
-		<div
-			className="section-item label"
-			style={{
-				backgroundColor: labelColor.backgroundColor,
-				color: labelColor.textColor,
-				borderColor: `${labelColor.borderColor}`,
-				paddingRight: canDelete ? '2px' : '8px'
-			}}
-		>
-			{name}{label.children}
-		</div>
+		<li>{name}{label.children}</li>

```

Comment:
I just remembered that this `Label` component is also used in the PR description (the full editor that shows all the details of an existing PR). We shouldn't make changes here that will also impact that view.

For now, can you just duplicate the `Label` component in this file and call it `LabelCreate`? Then we can safely modify `LabelCreate`.

Reply:
Fixed.